src/components/diagrams/singleLineDiagram/single-line-diagram.js

```diff
@@ -403,7 +416,15 @@ const SingleLineDiagram = forwardRef((props, ref) => {
         } else {
             setSvg(NoSvg);
         }
-    }, [props.svgUrl, forceState, snackError, intlRef, sldId]);
+    }, [
+        props.svgUrl,
+        forceState,
+        snackError,
+        intlRef,
+        sldId,
+        currentNode,
+        notificationIdList,
+    ]);
```

Comment:
If we use isNodeInNotificationList as a dependency instead of notificationIdList, could this will prevent a reload of all the SLD each time someone else works on another unrelated node ?

Reply:
Done